src/JustEat.StatsD/IStatsDTcpClient.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace JustEat.StatsD
+{
+    public interface IStatsDTcpClient : IStatsDClient
+    {
+    }
```

Comment:
The IStatsDClient (old IStatsDUdpClient) was public, therefore I wanted to stay consistent in case someone had taken a reference to it.